src/toolbox/toolbox_opmode.ts

```diff
@@ -0,0 +1,30 @@
+ import * as common from './toolbox_common'
+ import {category as robotCategory} from './/robot_category';
+ 
+ export function getToolboxJSON(
+     shownPythonToolboxCategories: Set<string> | null) {
+ 
+     return {
+         kind: 'categoryToolbox',
+         contents: [
+            robotCategory,
+            { kind: 'sep' },
+            ...common.getToolboxItems(shownPythonToolboxCategories)
+         ]
+     };
+ }
+ 
+ /*if (opt_robotBlocks.length) {
```

Comment:
Add a comment saying why this is commented out. Something like...
TODO: When editing an opmode, the toolbox should show blocks for calling functions that the user has defined in the robot module. This commented-out code will probably need to be reworked.